src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -898,7 +871,7 @@ class LicenceService(
         additionalConditionData = updatedAdditionalConditionData,
         additionalConditionUploadSummary = updatedAdditionalConditionUploadSummary,
       )
-    } as MutableList<AdditionalCondition>
```

Comment:
This never needed to be a mutable list, it just previously needed to have a mutable reference